### ISSUE TITLE
Wrong behaviour in the case statement

### ISSUE THREAD

**Issue by GISED-Link** [Barteling/STweep#36]
**Describe the bug**
When I format my code, line of code are deleted, see result below.

**Expected behavior**
 - The initial code before formatting.
```
Code shoud be formated correctly or at least not be formated at all
CASE state OF 
100:
    IF _MoverList[1]^.MoveToStation().MoverAxis^.isInPos THEN
        _MoverList[1]^.Station.Process.StartProcess();
        step := 101;
    END_IF

101:
    IF _MoverList[1]^.Station.Process.xDone THEN
		_MoverList[1]^.Station.Process.StopProcess();
		_MoverList[1]^.Station REF=0;
   		step := 199;
	END_IF
// wait until the next spot is free
199:
    // here we assume that the next spot is already free
    _MoverList[1]^.SetStation(_StationList[station2]);
    step := 200;
200:
    IF _MoverList[1]^.MoveToStation().MoverAxis^.isInPos THEN
        _MoverList[1]^.Station.Process.StartProcess();
        step := 201;
    END_IF
	end_case
```
 - The current result after formatting.
```
CASE state OF

100:
    IF _MoverList[1]^.MoveToStation().MoverAxis^.isInPos THEN
        _MoverList[1]^.Station.Process.StartProcess();
        step := 101;
    END_IF

101:
    IF _MoverList[1]^.Station.Process.xDone THEN
		_MoverList[1]^.Station.Process.StopProcess();
_MoverList[1]^.
Station REF=
0;
    step := 199;
// wait until the next spot is free
199:
    // here we assume that the next spot is already free
    _MoverList[1]^.SetStation(_StationList[station2]);
    step := 200;
200:
    IF _MoverList[1]^.MoveToStation().MoverAxis^.isInPos THEN
        _MoverList[1]^.Station.Process.StartProcess();
        step := 201;
    END_IF
end_case
```
 - The expected result.
```
CASE state OF 

100:
    IF _MoverList[1]^.MoveToStation().MoverAxis^.isInPos THEN
        _MoverList[1]^.Station.Process.StartProcess();
        step := 101;
    END_IF

101:
    IF _MoverList[1]^.Station.Process.xDone THEN
		_MoverList[1]^.Station.Process.StopProcess();
		_MoverList[1]^.Station REF=0;
   		step := 199;
	END_IF
// wait until the next spot is free
199:
    // here we assume that the next spot is already free
    _MoverList[1]^.SetStation(_StationList[station2]);
    step := 200;
200:
    IF _MoverList[1]^.MoveToStation().MoverAxis^.isInPos THEN
        _MoverList[1]^.Station.Process.StartProcess();
        step := 201;
    END_IF
END_CASE
```
 - An export of the used settings.
see file atteched (sorry, github doesn't support .stweep file)
[GISED_Link_STWEEP_config.zip](https://github.com/Barteling/STweep/files/6263593/GISED_Link_STWEEP_config.zip)

**STweep version info**
STweep for TwinCAT 3.3.0-Beta

**IDE version info**
TcXaeShell Version 15.0.28010.2050 D15.8
Microsoft .NET Framework Version 4.8.04084
Installed Version: Express
...
NuGet Package Manager   4.6.0
NuGet Package Manager in Visual Studio. For more information about NuGet, visit http://docs.nuget.org/.
...
STweepPackage Extension   1.0
STweepPackage Visual Studio Extension Detailed Info
...
TwinCAT XAE PLC   3.1.0.0
PLC Environment
Microsoft Visual Studio Community 2019 Version 16.8.5

**Additional context**
I tried the code in a function and I can reproduce the error

**Comment by Barteling** [comment]
Thanks for reporting,
Ill check what is going on here

Update, I can reproduce it. I'll prepare an hotfix tonight.

**Comment by Barteling** [comment]
Fixed in STweep for TwinCAT 3.3.2 ->  https://www.stweep.com/releases/
Codesys update will follow soon.
